### PR TITLE
DOC: mention altair_widgets in documentation

### PR DESCRIPTION
**Pull request by stsievert**
Mentions package `altair_widgets` in documentation and shows GIFs of examples.

I would add an example in the HTML but #306 is blocking tests.

### CODE REVIEW

File: doc/documentation/interactive.rst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+.. image:: images/cars-basic.gif
+
+It also supports different options for each encoding. Below, we show a subset
+of the advanced features:
+
+.. image:: images/cars-adv.gif
+
+An example notebook with these features is available on Binder:
+
```
Comment (jakevdp) [inline doc/documentation/interactive.rst]: Is there a link missing here?

Comment (stsievert) [inline doc/documentation/interactive.rst]: There *was* but it's removed now.

**Comment by stsievert** [comment]
I include GIFs of basic interactivity and changing some options.

Note: I tried to embed an altair widget in Sphinx documentation but ran into https://github.com/ipython/ipywidgets/pull/1213. I've also tried binder but ran into https://github.com/binder-project/binder/issues/94

**Comment by ellisonbg** [comment]
In this repo:

https://github.com/altair-viz/jupyterlab_voyager

We are working on a much more flexible and powerful UI for Vega-Lite in JupyterLab using Voyager. Do you think it makes sense to continue with the altair_widgets in light of that?

I do think we want a ipywidget based Vega-Lite renderer though - see #333

**Comment by stsievert** [comment]
> a much more flexible and powerful UI for Vega-Lite in JupyterLab using Voyager

Wow – that's a tool I'll likely use! I tried to install it to see how it looks (but ran into system specific bugs).

altair_widgets hasn't required much development time, and I'll continue developing as I need it. But we'll see how Voyager influences that...

**Comment by ellisonbg** [comment]
OK I am fine with us merging this PR in the meantime, with one change - can you add language that emphasizes that the widgets package is experimental?

File: doc/documentation/interactive.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.. _interactive-reference:
+
+Interactive visualization
+-------------------------
+
+Another package, `altair_widgets`_ provides interactive visualization of a
```
Comment (ellisonbg) [inline doc/documentation/interactive.rst]: experimental package

**Pull request merged**